tsconfig.json

```diff
@@ -1,5 +1,31 @@
 {
-  "extends": "./tsconfig.base.json",
-  "include": ["test/**/*.test.ts", "test/**/*.test.tsx"],
-  "exclude": ["node_modules"]
+  "compilerOptions": {
```

Comment:
Moved in here (unmodified) from `tsconfig.base.json`, which was not used anywhere else. The last other usage was deleted in #64043.